crates/sui-oracle/src/config.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
+use std::collections::HashMap;
+use std::net::SocketAddr;
+use std::time::Duration;
+use sui_config::Config;
+use sui_types::base_types::ObjectID;
+
+#[serde_as]
```

Comment:
`#[serde_as]` is not needed, we are not using serde_as to customise serde for the fields.

same for all other structs in this file.